README.md

```diff
@@ -90,6 +90,7 @@ Validation error: Number must be greater than 0 at "id"; Invalid email at "email
 - [isValidationErrorLike(error)](#isvalidationerrorlike)
 - [fromZodIssue(zodIssue[, options])](#fromzodissue)
 - [fromZodError(zodError[, options])](#fromzoderror)
+- [fromError(error[, options])](#fromerror)
```

Comment:
[question] @jmike @nikoskalogridis do you think we should move this documentation higher since it's the one we want to promote more over `fromZodError` and `fromZodIssue`?

Reply:
Yes, please do.

Reply:
Done ✅